src/Parser/Rogue/Subtlety/Modules/Features/ShadowDance.js

```diff
@@ -41,6 +50,23 @@ class ShadowDance extends Analyzer {
       this.inShadowDance = false;
     }
   }
+
+  statistic() {
+    const shadowBladesUptime = this.combatants.selected.getBuffUptime(SPELLS.SHADOW_BLADES.id) / this.owner.fightDuration;
```

Comment:
It looks like you mixed up class name and statistic? Class is called ShadowDance, this is getting Shadow Blades uptime.

Reply:
fixed